### DISCUSSION TITLE
invite fruit FB group?

### DISCUSSION THREAD

**Discussion by cocokola** [TheLazyGeekGuy/DIY-Waves-Soundgrid-Server#8] [General]
I searched FB and I'm on the forum thread that started all this, but I haven't heard of a group on FB.  Can I get an invite or point me where to sign up it would be fantastic.

I'm disabled and broke, did 30 years live work and putting together budget studio to do mixing and some mastering.  

If not kosher no worries I'll delete this.  Thx

**Comment by TheLazyGeekGuy** [comment]
Read it again : https://github.com/TheLazyGeekGuy/DIY-Waves-Soundgrid-Server

**Comment by cocokola** [comment]
Thanks.  I read the whole thing and it doesn't jump off the page you're was
where to join.

More importantly, this is what I get, hence my question.

**Comment by TheLazyGeekGuy** [comment]
![DIY Hyperlink](https://user-images.githubusercontent.com/35296076/218993611-0838cae9-2335-4f6d-a396-e2cb8ee4c8ec.PNG)

Hyperlinks does the magic !
